frontend/src/pages/learn/index.tsx

```diff
@@ -158,60 +177,88 @@ const Learn: FC = () => {
               }) => {
                 return (
                   <>
-                    {renderComponent()}
-                    {currentStepIndex === 0 && (
-                      <Button
-                        className="w-full bg-primary-700 p-4 text-center font-display text-base normal-case text-white hover:bg-primary-800"
-                        onClick={handleNext}
-                        disabled={isNextDisabled}
-                      >
-                        {t('quiz.navigation.start')}
-                      </Button>
-                    )}
-                    {currentStepIndex != 0 && (
-                      <div>
-                        <MobileStepper
-                          variant="progress"
-                          steps={10}
-                          position="static"
-                          activeStep={currentStepIndex}
-                          backButton={undefined}
-                          nextButton={undefined}
-                          classes={{
-                            progress: 'w-full',
-                          }}
-                        />
-                        <p className="text-center">{currentStepIndex} of 9</p>
+                  {showConfirmation ? (
```

Comment:
This one is open for debate.. but with so much JSX in each branch of this ternary, would it make sense to extract them out into private components?

This would be much easier to read: `{showConfirmation ? <OneElement /> : <OtherElement />}`

Again, this one is definitely subjective, so maybe someone else can either agree or disagree (I'm fine either way).